### PR TITLE
Documentation/addition of required files

### PR DESCRIPTION
**Pull request by ayato91**
@immadisairaj  All the requested changes are live now :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This will give you any eventual conflicts and allow you to easily solve them in
 
 ## Setup Guide
 
-This app is written in Flutter. Even though Flutter is cross-platform, This app is currently focused on Android or iOS. So in order to setup and run the project locally, you need to have an emulator(android or iOS) or a real device.
+This app is written in Flutter. Even though Flutter is cross-platform, This app is currently focused on Android or iOS. So in order to setup and run the project locally, you need to have `an emulator(android or iOS) or a real device` and `flutter installed` locally.
```
Comment (immadisairaj) [inline CONTRIBUTING.md:78]:
```suggestion
This app is written in Flutter. Even though Flutter is cross-platform, This app is currently focused on Android or iOS. So in order to setup and run the project locally, you need to have an emulator(android or iOS) or a real device and flutter installed locally.
```
No need of those ..

**Review by immadisairaj — CHANGES_REQUESTED** [review]
Just one small change and all good to go. Thanks for the contribution.

**Comment by ayato91** [comment]
@immadisairaj  Done :)

**Review by immadisairaj — APPROVED** [review]
LGTM

**Comment by ayato91** [comment]
@immadisairaj Can you put a hacktoberfest accepted tag?

**Pull request merged**